NAMESPACE

```diff
@@ -1,65 +1,17 @@
 # Generated by roxygen2: do not edit by hand
 
-
```

Comment:
These changes seem significant? Is this ok to change?

Reply:
The "Writing R Extensions" manual's guidance is to only use S3method() to register S3 method (see https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Registering-S3-methods). roxygen2 documentation specifies that the generics need to be explicitly imported using the `@importFrom` tag (see https://roxygen2.r-lib.org/articles/namespace.html?q=rawNamespace#imports). So those changes follow the guidelines to clean up old out-of-date NAMESPACE sections. Also, given R 3.6.0 was released in Apr 2019 (more than 3 years ago), users who still use older versions of R should be very rare. Their R experiences are likely to be buggy not because of the NAMESPACE exports here but due to the staleness of their environment.